src/frontend/src/lib/services/address.services.ts

```diff
@@ -30,21 +34,23 @@ import { get } from 'svelte/store';
 const loadTokenAddress = async <T extends Address>({
 	tokenId,
 	getAddress,
-	setIdbAddress
+	setIdbAddress,
+	addressStore
 }: {
 	tokenId: TokenId;
```

Comment:
We still need the tokenId here?

Reply:
yes, but only for the error message

```typescript
toastsError({
	msg: {
		text: replacePlaceholders(get(i18n).init.error.loading_address, {
			$symbol: tokenId.description ?? ''
		})
	},
	err
});
```

i would prefer not to generalize the error message, but WDYT?

Reply:
Makes sense. Maybe we can pass the `tokenSymbol` as parameter in another PR or something, just thinking out at loud. When I read `tokenId` passed to a function I automatically think that it will be use for some feature but, that's maybe just me. All good. Thanks for the feedback.